components/Main/Main.tsx

```diff
@@ -37,5 +37,5 @@ const Main: React.FunctionComponent<MainProps> = ({ i18n }) => {
     </main>
   );
 };
-
+Main.whyDidYouRender = false;
```

Comment:
`why-did-you-render` will notify as for rerenders of `useUser` when this is set to `true`